cfgov/v1/models/images.py

```diff
@@ -84,10 +84,19 @@ def resize(self, size):
         operation.run(mock_image, image=None, env={})
         return mock_image.width, mock_image.height
 
+    @property
```

Comment:
I thought all this did was stick a default value into the editor when you're adding an image embed to a rich text block. The Wagtail user still has the ability to modify this to be whatever they want, or blank. It doesn't actually change what gets called on page render.